sky/data/storage.py

```diff
@@ -312,6 +310,68 @@ def mount_command(self, mount_path: str) -> str:
         """
         raise NotImplementedError
 
+    def csync_command(self,
```

Comment:
usually, we won't write a function in the base class and override it to `NotImplemented` in subclass, which seems strange. could we consider changing this function name to `_csync_command` and

```python
# in base class
def csync_command(...):
    raise NotImplemented
# in GCS, S3
def csync_command(...):
    return _csync_command(...)
# in IBM, R2: nothing
```

cc @Michaelvll for a look here; not sure if it is the best practice

Reply:
It is from me. But what is the relation between the two comments..? I think they refer to different things. Here I mean to change the function name `csync_command` to `_csync_command` (i.e. add a leading underscore) and call it from subclass, not to separate them.